app/models/peep.rb

```diff
@@ -0,0 +1,34 @@
+class Peep
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :content, Text, :required => true
+  property :created_at, EpochTime
+  property :user_id, Integer
+
+  belongs_to :user
+
+  def how_long_ago
+    time = seconds_difference.to_i
+
+    case time
+      when 0 then 'just now'
+      when 1 then 'a second ago'
+      when 2..59 then time.to_s+' seconds ago'
+      when 60..119 then 'a minute ago' #120 = 2 minutes
+      when 120..3540 then (time/60).to_i.to_s+' minutes ago'
+      when 3541..7100 then 'an hour ago' # 3600 = 1 hour
+      when 7101..82800 then ((time+99)/3600).to_i.to_s+' hours ago'
+      when 82801..172000 then 'a day ago' # 86400 = 1 day
```

Comment:
Style/CaseIndentation: Indent `when` as deep as `case`. (https://github.com/bbatsov/ruby-style-guide#indent-when-to-case)<br>Style/NumericLiterals: Separate every 3 digits in the integer portion of a number with underscores(_). (https://github.com/bbatsov/ruby-style-guide#underscores-in-numerics)